CMakeLists.txt

```diff
@@ -6,10 +6,10 @@ project(SkeletalRepresentation)
 # Extension meta-information
 set(EXTENSION_HOMEPAGE "http://slicer.org/slicerWiki/index.php/Documentation/Nightly/Extensions/SkeletalRepresentation")
 set(EXTENSION_CATEGORY "Topology, Skeletonization")
-set(EXTENSION_CONTRIBUTORS "Jared Vicory (Kitware, Inc.), Pablo Hernandez-Cerdan (Kitware, Inc.), Beatriz Paniagua (Kitware, Inc.)")
-set(EXTENSION_DESCRIPTION "Collection of Skeletal Model Extensions")
-set(EXTENSION_ICONURL "http://www.example.com/Slicer/Extensions/SkeletalRepresentation.png")
-set(EXTENSION_SCREENSHOTURLS "http://www.example.com/Slicer/Extensions/SkeletalRepresentation/Screenshots/1.png")
+set(EXTENSION_CONTRIBUTORS "Zhiyuan Liu (University of North Carolina at Chapel Hill), Junpyo Hong (University of North Carolina at Chapel Hill), Stephen M. Pizer (University of North Carolina at Chapel Hill), Jared Vicory (Kitware, Inc.), Pablo Hernandez-Cerdan (Kitware, Inc.), Beatriz Paniagua (Kitware, Inc.), Jean-Christophe Fillion-Robin (Kitware, Inc.)")
+set(EXTENSION_DESCRIPTION "Toolkit that provides users to establish, refine, and visualize skeletal representation of an object.")
+set(EXTENSION_ICONURL "http://slicer.org/slicerWiki/index.php/Documentation/Nightly/Extensions/SkeletalRepresentation.png")
+set(EXTENSION_SCREENSHOTURLS "https://github.com/ZhiyLiu/SRepExtension/blob/master/visualize_s_rep.png")
```

Comment:
So this would be:

```
 set(EXTENSION_SCREENSHOTURLS "https://github.com/KitwareMedical/SRepExtension/blob/master/visualize_s_rep_NEW_NAME.png") 
```